src/Sylius/Bundle/AdminBundle/templates/shared/crud/common/content/form.html.twig

```diff
@@ -1,11 +1,15 @@
+{% if form is not defined %}
+    {% set form = hookable_metadata.context.form %}
+{% endif %}
```

Comment:
```suggestion
{% if form is not defined %}
    {% set form = hookable_metadata.context.form %}
{% endif %}

{% if resource is not defined %}
    {% set resource = hookable_metadata.context.resource %}
{% endif %}
```

Reply:
I've tried this change there and that works
https://github.com/Sylius/Stack/pull/120